shared/js/injected-content-scripts/fingerprint-protection.js

```diff
@@ -197,14 +197,39 @@
         }
     }
 
+    async function getSjcl () {
+        const url = chrome.runtime.getURL('/data/sjcl.js')
```

Comment:
I think bundling this in the build script will be faster and also cleaner than doing it at injection time.